tests/integration/test.design_docs.js

```diff
@@ -96,6 +96,39 @@ adapters.forEach(function (adapter) {
       });
     });
 
+    it('Indexing event', function (done) {
+      var docs1 = [
+        doc,
+        {_id: 'dale', score: 3},
+        {_id: 'mikeal', score: 5},
+        {_id: 'max', score: 4},
+        {_id: 'nuno', score: 3},
+      ];
+      var db = new PouchDB(dbs.name);
+      // Test invalid if adapter doesnt support mapreduce
+      if (!db.query || adapter !== 'local') {
+        return done();
+      }
+
+      let indexingEvents = [];
+
+      db.on('indexing', function (result) {
+        indexingEvents.push(result);
+      });
+
+      db.bulkDocs({ docs: docs1 }, function () {
```

Comment:
Its a slow process however we have started moving over to using `await` for the tests instead of using the callbacks and `done` so new tests should probably do the same

Reply:
Thank you, done! Please note that we had to upgrade eslint `ecmaVersion` in order to use arrow functions and async/await syntax